files/packages/__name__/.eslintrc.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = {
+  root: true,
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    ecmaFeatures: {
+      legacyDecorators: true,
+    },
+  },
+  plugins: ['ember'],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended',
+    'plugin:prettier/recommended',
+  ],
+  env: {
+    browser: true,
+  },
+  rules: {},
+  overrides: [
+    // node files
+    {
+      files: [
+        './.eslintrc.js',
```

Comment:
`./*.js`

Reply:
I think I based this on the default addon blueprint, which lists files explicitly. Don't why, as your suggestion would probably also work (given that all browser modules are in `src/`)

Reply:
yeah, this is the technique I use to simplify my configs over here: https://github.com/NullVoxPopuli/eslint-configs/blob/main/configs/ember.js#L129

that way you don't need to alter your lints _every time_ you add a new top-level js file